extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -536,204 +614,160 @@ class="pi-horizontal-group-item pi-data-value pi-font pi-border-color pi-item-sp
 									<p>Links</p>
 								</nav>
 							</aside>',
-				'description' => 'Infobox with navigation'
+				'description' => 'Infobox with navigation',
+				'mockParams' => [ ],
+				'accentColor' => '',
+				'accentColorText' => ''
 			],
+			// horizontal group tests
 			[
 				'input' => [
 					[
-						'type' => 'image',
+						'type' => 'group',
 						'data' => [
-							'alt' => 'image alt',
-							'url' => 'http://image.jpg',
-							'ref' => 1,
-							'name' => 'test1',
-							'key' => 'test1',
-							'isVideo' => false
+							'value' => [
+								[
+									'type' => 'data',
+									'data' => [
+										'label' => 'test label 1',
+										'value' => 'test value 1'
+									]
+								],
+								[
+									'type' => 'data',
+									'data' => [
+										'label' => 'test label 2',
+										'value' => 'test value 2'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "test value 2" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)